frontends/mit-open/src/page-components/Header/Header.tsx

```diff
@@ -7,48 +7,137 @@ import {
   NavDrawer,
   Toolbar,
   ClickAwayListener,
+  ActionButtonLink,
 } from "ol-components"
+import { RiSearch2Line } from "@remixicon/react"
 import { MITLogoLink, useToggle } from "ol-utilities"
 import UserMenu from "./UserMenu"
 import { MenuButton } from "./MenuButton"
 import {
   DEPARTMENTS,
   RESOURCE_DRAWER_QUERY_PARAM,
+  SEARCH,
   querifiedSearchUrl,
 } from "@/common/urls"
 import { useSearchParams } from "react-router-dom"
+import { useUserMe } from "api/hooks/user"
 
-const Bar = styled(AppBar)`
-  height: 56px;
-  padding: 0 8px;
-  z-index: ${({ theme }) => theme.zIndex.drawer + 1};
-  background-color: ${({ theme }) => theme.custom.colors.white};
-  color: ${({ theme }) => theme.custom.colors.black};
-  display: flex;
-  flex-direction: column;
-  box-shadow: 0 2px 10px rgba(120 169 197 / 15%);
-`
+const Bar = styled(AppBar)(({ theme }) => ({
+  height: "80px",
+  padding: "0 8px",
+  zIndex: theme.zIndex.drawer + 1,
```

Comment:
@gumaerc As mentioned in slack, the learning resource drawer is currently obscured by the header:
<img width="400" alt="Screenshot 2024-05-21 at 10 26 37 AM" src="https://github.com/mitodl/mit-open/assets/9010790/dd4e7d58-7aaf-4815-be86-d57097d7b36b">

since we DO want the learning resource drawer to be OVER the header, my recommendation is:
- do not override the zIndex here
- instead change the NavDrawer to have `zIndex = header + 1`. (Might be on `PaperProps`, I'm not sure. Check the CSS in dev console.)

It probably is still worthwhile storing the header height in the theme (e.g., `theme.dimensions.headerHeader`) or as a constant.